docs/development/writing_zeppelin_interpreter.md

```diff
@@ -207,6 +208,17 @@ You can make the dynamic form in the notebook not trigger execution after select
 }
 ```
 
+
+### Check if the paragraph is empty before running (Optional)
+The notebook's paragraph defaults will not run if it is empty.
+You can set `config.checkEmpty=false`, Run when the paragraph of the notebook is empty.
```

Comment:
`Run when the paragraph of the notebook is empty` -> `to run even when the paragraph of the notebook is empty`?

Reply:
Ok, I am going to change it. Thank you!